components/inputs/input-date.js

```diff
@@ -313,20 +316,25 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 	_handleDropdownClose() {
 		this._calendar.reset();
 		this._dropdownOpened = false;
+		this._textInput.scrollIntoView({block: 'nearest', behavior: 'smooth', inline: 'nearest'});
 	}
 
 	_handleDropdownOpen() {
-		this.shadowRoot.querySelector('d2l-focus-trap').scrollIntoView({block: 'nearest', behavior: 'smooth', inline: 'nearest'});
+		if (!this._dropdown.openedAbove) this.shadowRoot.querySelector('d2l-focus-trap').scrollIntoView({block: 'nearest', behavior: 'smooth', inline: 'nearest'});
```

Comment:
Scroll to the `input-text` since it has focus on initial dropdown opening. Don't scroll to show `focus-trap` if opened above since that results in jumpiness (scroll jumps to top of calendar dropdown content, then jumps down to input-text)